mibs/fortinet/IPV6-TC

```diff
@@ -0,0 +1,67 @@
+IPV6-TC DEFINITIONS ::= BEGIN
```

Comment:
IPV6-TC is a standard mib, it shouldn't be included here (it is already in the base directory)